contracts/wrappers/libraries/Deployable.ts

```diff
@@ -113,11 +115,11 @@ export const builder = {
 
 export abstract class Opcodes {
   static initialize = 0xba466447
-  static initializeAndSend = 0xe95e1156
+  static initializeAndSend = 0xb0ec5157
```

Comment:
we should compute these with `crc32` utility vs. hardcoding in lib code, and we can verify hardcoded (visual) values in a test